plaidml/edsl/edsl_doc.py

```diff
@@ -0,0 +1,261 @@
+# Copyright 2020 Intel Corporation.
+# DO NOT TOUCH THIS FILE
```

Comment:
Drop this warning, also update the comments to reflect the latest changes to the docs